pol-core/pol/tooltips.cpp

```diff
@@ -96,12 +95,14 @@ void SendAOSTooltip( Network::Client* client, UObject* obj, bool vendor_content
   if ( obj->isa( UOBJ_CLASS::CLASS_CHARACTER ) )
   {
     Mobile::Character* chr = (Mobile::Character*)obj;
-    desc = ( !chr->has_title_prefix() ? " \t" : chr->title_prefix() + " \t" ) + chr->name() +
-           ( !chr->has_title_suffix() ? "\t " : "\t " + chr->title_suffix() );
+
+    desc = fmt::format( "{} \t{}\t {}", chr->title_prefix(), chr->name(), chr->title_suffix() );
     if ( chr->has_title_race() )
-      desc += " (" + chr->title_race() + ")";
+      desc += fmt::format( "{}({})", chr->has_title_suffix() ? " " : "", chr->title_race() );
```

Comment:
Makes sense, only need to preface with `" "` if there is a title suffix. If no title suffix, it will already have a trailing `" "`